lib/client-assets.php

```diff
@@ -296,9 +296,17 @@ function gutenberg_register_scripts_and_styles() {
 			"\n",
 			array(
 				'( function() {',
-				'	if ( wp && wp.components && wp.serverSideRender && ! wp.components.ServerSideRender ) {',
-				'		wp.components.ServerSideRender = wp.serverSideRender;',
-				'	};',
+				'	if ( wp && wp.components && wp.serverSideRender ) {',
+				'		wp.components.ServerSideRender = wp.element.forwardRef( ( props, ref ) => {',
```

Comment:
We can't use arrow function syntax here, it will certainly break in IE11.

The `Object.assign` later is okay because it's polyfilled, but we can't polyfill these syntaxes.